integration-tests/wsdl2java/src/main/resources/application.properties

```diff
@@ -1,2 +1,2 @@
 quarkus.cxf.codegen.wsdl2java.includes = **.wsdl
-quarkus.cxf.codegen.wsdl2java.additional-params = -wsdlLocation,classpath:wsdl/CalculatorService.wsdl,-xjc-Xts
+quarkus.cxf.codegen.wsdl2java.additional-params = -wsdlLocation,classpath:wsdl/CalculatorService.wsdl,-xjc-Xbg,-xjc-Xdv,-xjc-Xjavadoc,-xjc-Xproperty-listener,-xjc-Xts,-xjc-Xwsdlextension
```

Comment:
`-xjc-Xdv` is not covered by any assertion in the test below. I wanted to use the default value of 42 in Result.result, but then, after I published the test container image 1.3, I found that it does not work for primitive types. 
But at least, it is passed to wsdl2java and wsdl2java does not complain.